README.md

```diff
@@ -5,6 +5,9 @@
 In this repository you will find our Sauce Labs sample-app. You can use it as a sample app for testautomation on your local machine or in our Real Device Cloud.
 The latest version of the iOS and Android app can be found [here](https://github.com/saucelabs/sample-app-mobile/releases)
 
+> **NOTE<br>**
+> Make sure to use yarn instead of NPM. NPM gives a lot of problems currently with React Native. How to setup yarn can be found [here](https://yarnpkg.com/lang/en/docs/install/#mac-stable)
```

Comment:
What kinds of problems? And where should we look to know when they are resolved and we can start using NPM again?

Reply:
Good remarks and questions. I don't have exact answers, only from what I've seen with the migration.

- React Natve 0.60.x now uses Pods for linking dependencies, from what I've seen with NPM is that is messes up this linking 9 out of 10 times
- Installing other dependencies are mostly recommended to install with yarn (not only for React Native but also with ReactJS)
- It costed me around a day to get everything working with NPM, but with yarn I didn't had any problems migrating to RN 0.60
- yarn is faster

and I can go on

Reply:
Interesting - good to know! I think it would make sense to expand upon the notes above in the README since npm is often the default choice for building Node projects - I think you can basically copy/paste what you said above into the doc and it'll make things a lot clearer for the reader (and also the note about .gitinore'ing package-lock.json so it's clear there's a particular reason for that choice)

Reply:
Will do, again thanks for your time, appreciate it a lot!